crates/cargo-contract/src/cmd/extrinsics/integration_tests.rs

```diff
@@ -309,3 +309,94 @@ async fn build_upload_remove() {
     // prevent the node_process from being dropped and killed
     let _ = node_process;
 }
+
+/// Sanity test the whole lifecycle of:
+///   new -> build -> upload -> instantiate -> info
+///
+/// # Note
+///
+/// Requires [substrate-contracts-node](https://github.com/paritytech/substrate-contracts-node/) to
+/// be installed and available on the `PATH`, and the no other process running using the default
+/// port `9944`.
+///
+/// ** This test is ignored for now since the substrate-contracts-node is not installed on CI **
+/// It will be addressed in a follow up PR, for now it can be run locally by commenting out the
+/// `ignore` attribute below
+#[async_std::test]
+async fn build_upload_instantiate_info() {
+    init_tracing_subscriber();
+
+    let tmp_dir = tempfile::Builder::new()
+        .prefix("cargo-contract.cli.test.")
+        .tempdir()
+        .expect("temporary directory creation failed");
+
+    // Spawn the contracts node
+    let node_process = ContractsNodeProcess::spawn(CONTRACTS_NODE)
+        .await
+        .expect("Error spawning contracts node");
+
+    tracing::debug!(
+        "Creating new contract in temporary directory {}",
+        tmp_dir.path().to_string_lossy()
+    );
```

Comment:
```suggestion
```

For tests, we don't include tracing output. It's fine to have this during development or for debugging, but shouldn't be in `master`.